### ISSUE TITLE
Issues importing wallet 

### ISSUE THREAD

**Issue by WorldsWorstInvestor** [ethereum/mist#3003]
Hi,
I am currently attempting to import my old wallet onto a new pc in order to be able to transfer the 2 ethereums that I have stored on it onto an exchange, but can't for the life of me figure out how to do so.

I have tried the command promt fix recommended here : https://ethereum.stackexchange.com/questions/465/how-to-import-a-plain-private-key-into-geth-or-mist?noredirect=1&lq=1 But am met with the following" 'geth' is not recognized as an internal or external command,operable program or batch file. "

Is it referring to a command prompt inside mist that I'm not aware of? The one I used is the regular old ms command promp. If so where is this prompt located?

I have access to both my public and private key, the folder called keystore, and the password that I created for the account. What I don't have is any programming knowledge to speak of. 

So if someone could please, explain, in laymans terms, a way for me to either import my wallet, or transfer my ether from the wallet to the exchange without accessing it through mist, that would be very much appreciated.

old wallet address : 0xf9FA047336bCfE8cC290310423c4CC509Fc99765 (where my ether is located)
new wallet address (etherbase account created upon download of mist onto new pc, which i currently have access to): 0xc5F77607AA3283Bbb60150f7F8E3FcEfBdaEB1F7

Thank you for your time, and for helping out a hapless noob who dipped his head in too deep!
--

**Comment by sandervanlaar** [comment]
To transfer your ETH without Mist, you can use the following guide: https://www.cryptocompare.com/wallets/guides/how-to-use-myetherwallet/

I haven't tried it yet, but Step 5 looks promising.
